app/services/fetch_csv_service.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require 'net/http'
+require 'byebug'
```

Comment:
will this work on prod env?

Reply:
No, I should move the `open-uri` and `net-http` in Gemfile, and remove byebug...